crypto/bn/bn_print.c

```diff
@@ -286,6 +289,7 @@ int BN_asc2bn(BIGNUM **bn, const char *a)
     return 1;
 }
 
+#ifndef FIPS_MODE
```

Comment:
It could be pulled in if needed by making BIO_snprintf something that gets passed to the provider.

Reply:
I'm hoping we won't need any of these functions.